packages/anoncreds-rs/src/services/AnonCredsRsVerifierService.ts

```diff
@@ -24,27 +24,15 @@ export class AnonCredsRsVerifierService implements AnonCredsVerifierService {
         rsSchemas[schemaId] = schemas[schemaId] as unknown as JsonObject
       }
 
-      const revocationRegistryDefinitions: Record<string, RevocationRegistryDefinition> = {}
-      const lists = []
+      const revocationRegistryDefinitions: Record<string, JsonObject> = {}
+      const lists: JsonObject[] = []
 
       for (const revocationRegistryDefinitionId in revocationRegistries) {
         const { definition, revocationStatusLists } = options.revocationRegistries[revocationRegistryDefinitionId]
 
-        revocationRegistryDefinitions[revocationRegistryDefinitionId] = RevocationRegistryDefinition.fromJson(
-          definition as unknown as JsonObject
-        )
-
-        for (const timestamp in revocationStatusLists) {
-          lists.push(
-            RevocationStatusList.create({
-              issuerId: definition.issuerId,
-              issuanceByDefault: true,
-              revocationRegistryDefinition: revocationRegistryDefinitions[revocationRegistryDefinitionId],
-              revocationRegistryDefinitionId,
-              timestamp: Number(timestamp),
-            })
-          )
-        }
+        revocationRegistryDefinitions[revocationRegistryDefinitionId] = definition as unknown as JsonObject
```

Comment:
Any chance we can improve the casting? If we do `as unknown as...` it is very close to just using any and should be avoided whenever.

Reply:
Yes, I agree, it's something we are still missing in anoncreds-rs JS API. I think we'll need to complete AnonCreds object modeling in AFJ and replicate it in anoncreds-rs javascript wrapper so they match.